helper/schema/serialize.go

```diff
@@ -57,6 +57,12 @@ func SerializeValueForHash(buf *bytes.Buffer, val interface{}, schema *Schema) {
 				buf.WriteString(strconv.FormatFloat(innerVal, 'g', -1, 64))
 			case string:
 				buf.WriteString(innerVal)
+			case bool:
```

Comment:
Where are we using this and do we have a way to not do this change?  I assume, no, but I thought I would ask.